Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [workspace]
 members = [
+    "conversions",
```

Comment:
im confused, didnt we change this previously from "conversions" to "types"?

Reply:
We might, I like "conversions", as this library is mostly concerned with converting between `libsecp256k1` and `plonky2_ecdsa` types